backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -214,9 +220,29 @@ protected CreateSnapshotForVmParameters getCreateSnapshotParameters() {
         params.setDiskIdsToIgnoreInChecks(getImageGroupIds());
         params.setNeedsLocking(false);
         params.setEndProcedure(EndProcedure.COMMAND_MANAGED);
+
+        updateImageInitialSizeInBytes(params);
+
         return params;
     }
 
+    private void updateImageInitialSizeInBytes(CreateSnapshotForVmParameters params) {
+        if (FeatureSupported.isLiveStorageMigrationInitialSizeSupported(getVm())) {
+            StorageDomainStatic sourceDomain = storageDomainStaticDao.get(getParameters().getSourceStorageDomainId());
+            if (sourceDomain.getStorageType().isBlockDomain()) {
+                int volumeUtilizationChunkInMB = Config.getValue(ConfigValues.VolumeUtilizationChunkInMB);
+                long imageInitialSizeInBytes = 3 * SizeConverter.convert(volumeUtilizationChunkInMB,
+                        SizeConverter.SizeUnit.MiB,
+                        SizeConverter.SizeUnit.BYTES).longValue();
+                // TODO: do I need to create a fresh copy of the original disk here?
+                //DiskImage disk = DiskImage.copyOf(getDiskImage());
+                DiskImage disk = getDiskImage();
```

Comment:
I don't think there is a disk image for this command, and even if there was, we don't want to copy it - we want to create a new disk image  and set its initial size.